cirq/google/devices/known_devices_test.py

```diff
@@ -17,6 +17,7 @@
 import cirq
 import cirq.google as cg
 import cirq.google.common_serializers as cgc
+import cirq.google.devices.known_devices as cgdk
```

Comment:
is this how we're supposed to use it?

Reply:
Good question.  I wasn't sure if we wanted to expose it in the package for cirq.google or not.

Any opinions?

Reply:
I personally find the "use the first letter of each submodule" convention very obfuscating and would support any other convention that avoids it :) 

The rest of cirq operates under the "put everything in the top-level namespace" convention

Reply:
We should expose Sycamore23 from cirq.google, but not from top-level cirq, as it is a google-specific device.

`cirq.google.Sycamore23` same as `cirq.google.Bristlecone`.

Reply:
Done.